src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -105,15 +112,17 @@
         linux-arm64;
         " />
 
-      <Crossgen2SupportedRids Include="linux-musl-x64;linux-x64;win-x64" />
+      <Net50Crossgen2SupportedRids Include="linux-musl-x64;linux-x64;win-x64" />
+      <Crossgen2SupportedRids Include="@(Crossgen2SupportedRids)" />
```

Comment:
Should this be `<Crossgen2SupportedRids Include="@(Net50Crossgen2SupportedRids)" />`?